scripts/griddle.jsx

```diff
@@ -707,11 +708,11 @@ var Griddle = React.createClass({
                 hasMorePages={hasMorePages}
                 onRowClick={this.props.onRowClick}/></div>)
     },
-    getContentSection: function(data, cols, meta, pagingContent, hasMorePages){
+    getContentSection: function(data, extraData, cols, meta, pagingContent, hasMorePages){
```

Comment:
Also, don't mean to make things more complicated, but after thinking about it further I was wondering if it might make more sense to call the parameter globalData instead of extraData since it represents data that is global to the Griddle instance (doesn't change per row).  If you like that idea I can just create another completely separate PR and we can close this one.